module.json

```diff
@@ -6,8 +6,8 @@
   "authors": [],
   "url": "https://github.com/foundryvtt/world-anvil",
   "version": "1.3.3",
-  "minimumCoreVersion": "0.7.6",
-  "compatibleCoreVersion": "9",
+  "minimumCoreVersion": "10",
```

Comment:
There are some other manifest changes that should change for full V10 compatibility. Please see https://foundryvtt.com/article/manifest-migration-guide/

Reply:
Good to know.
I will work on that.